masterlist.yaml

```diff
@@ -9922,19 +9922,25 @@ plugins:
         name: 'Races of Nirn - Kapo''tun SE'
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/77612/'
         name: 'Races of Nirn - Kapotun Reborn'
+    group: *racesGroup
     inc:
+      - name: 'DWKaPoTunRace.esp'
+        condition: 'checksum("DWKaPoTunRace.esp", D34873CA)'
```

Comment:
What do you want to convey with this `inc` entry? As it is, you are saying that `DWKaPoTunRace.esp` is incompatible with itself (if the checksum is identical). Do you want to convey, that the plugin with that checksum is obsolete?

Reply:
Rather that one version shouldn't be changed to another. Reborn can't be replaced by original and vice versa, unless upgrade patch is used. May have been wrong message or format chosen.

Reply:
Update: deleted these lines.